json-to-go.js

```diff
@@ -16,7 +16,7 @@ function jsonToGo(json, typename)
 
 	try
 	{
-		data = JSON.parse(json);
+		data = JSON.parse(json.replace(/\.0/g, ".1"));
```

Comment:
Clever, but this could mess up object key names. `{"1.0": 5}` becomes `{"1.1": 5}`. Seems ludicrous, but I could see this being the case when dealing with version numbers or coordinates.